### DISCUSSION TITLE
开源软件的技术写作 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#20] [Comments]
# 开源软件的技术写作 | 夜天之书

开源社群虽然是围绕开源软件建立起来的，但是建设繁荣的开源生态，依靠只读的源代码是不够的。建设繁荣的开源生态需要开源协同，而协同的基础是沟通。沟通不仅有论坛和聊天室的交互式形式，还有主动的内容生产和发布，包括不同受众的文档的撰写和翻译，以及技术文章的传播等等。这些内容创造都可以归类为技术写作，也就是说，围绕开源软件的技术写作，是建设繁荣的开源生态的重要一环。 本文尝试讨论这些不同类型的技术写作在现实

https://tisonkun.org/2022/05/08/open-source-tech-writing/

**Comment by lanx-x** [comment]
我注意到博客使用自定义的字体 Noto Serif Simplified Chinese
这个不会导致下载体积庞大的字体文件吗?

**Reply by tisonkun** [reply]
我在 `gh-pages` 分支里没有看到相应的 otf 文件，我不太了解这方面的知识，可能是 CSS 引用了在线的资源（Google Font）。起码现在阅读浏览的体验没有明显的卡顿，至少用户层面没有看到问题。

**Reply by tisonkun** [reply]
不过确实之前在用 Materials For Mkdocs 的时候，碰到过因为要下载字体文件导致构建阶段时间比较长的情况，但是也不会影响浏览。

**Comment by shanksGuo** [comment]
博客有没有RSS地址，习惯使用RSS订阅但是没搜到你的RSS地址

**Reply by tisonkun** [reply]
噢，这个应该要加一下支持，你可以到 https://github.com/tisonkun/tisonkun.github.io/issues 开个 issue 我有时间加一下。

**Reply by tisonkun** [reply]
Fixed by 2ff462ea14a72870d94697a38073120cb9a613ef.

**Comment by czy-29** [comment]
阅读时间11分钟是认真的吗？😂这个时间是不是主题根据字数自动生成的？这字数小说11分钟可能还能读完，技术文章基本不太可能…

**Reply by tisonkun** [reply]
哈哈哈哈，确实如此。我也没仔细看 Hexo 或者 NexT 谁生成的这个组件是啥逻辑，你有时间或许可以拔下代码研究一下。

我自己看别人的万字长文基本是打开了放一边，等偶尔有大半个小时的时间再一口气看完。我自己也想着文章写短一点，但是总想把话讲全了，一个一个点写出来不小心就很长了。其实这篇文章对于“你为什么需要一个内容团队”和文档代码化等等主题都还没有展开呢。希望我能学会怎么精简内容。
